test/test_examples.py

```diff
@@ -310,6 +310,24 @@ def test_welford(self):
             )
         )
 
+    def test_low_mem_dropout(self):
```

Comment:
Test backwards (not just fwd) and assert that the same elements are dropped out in bwd as fwd (and different elements are dopped out if you change the seed).